Updated Council website (#23)
thunderbird/council-docs: council-website/docs/election.md

# Thunderbird Council Election

Join the Thunderbird Council – Help Shape the Future of Communication and More!

## Election Timeline
* First Monday of March: We will officially announce the electoral and open up the period for self-nominations. This will remain open for one week, inviting all eligible candidates to put forward their names for consideration.
* Following the close of nominations, we will enter a discussion period. This week will be dedicated to community engagement, allowing members to deliberate on the nominees' contributions and visions for the future.
* The voting process will commence, with each member having one week to cast their vote. 
* Upon the conclusion of the voting period, we will announce the new Council.  

## Why join the Thunderbird Council?
* Be Part of a Global Impact: Thunderbird is more than a communication client; it is a statement about the power of open-source and community-driven software.
* Contribute to Success: Your ideas and efforts can directly contribute to the ongoing development and success of Thunderbird.

> * Diverse Team Collaboration: Work with a dynamic team of 7 members from varied backgrounds – a vibrant tapestry of diverse perspectives and cultures

Reviewer: Vibrant tapestry sounds very AI generated. Also TBH I'm not sure how dynamic we are :D

```suggestion
* Diverse Team Collaboration: Work with a team of 7 members from varied backgrounds from across the world.
```
